src/store/actions.ts

```diff
@@ -0,0 +1,40 @@
+import { create } from 'zustand';
+import { auth } from "../firebase";
+import { createUserWithEmailAndPassword, signInWithEmailAndPassword } from 'firebase/auth';
+
+type AuthState = {
+  error: string | null;
```

Comment:
```suggestion
 LoginProps & 
```

удали поле error, он уже есть в файле src/pages/signIn/type.tsx